di_website/events/models.py

```diff
@@ -0,0 +1,88 @@
+from django.db import models
+from wagtail.admin.edit_handlers import (FieldPanel, InlinePanel,
+                                         MultiFieldPanel, StreamFieldPanel)
+from wagtail.core.fields import StreamField
+from wagtail.core.models import Page
+
+from di_website.common.base import OtherPage, StandardPage, get_paginator_range
+from di_website.common.blocks import BaseStreamBlock
+from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
+from django.contrib.contenttypes.models import ContentType
+
+from taggit.models import Tag, TaggedItemBase
+
+""" Content of each Event that makes up events list """
+
+
+class EventPage(StandardPage):
+
+    template = "events/events_page.html"
+
+    start_time = models.DateTimeField()
```

Comment:
You're going to require 4 properties... 2 for dates (start_date, end_date), and 2 for time (start_time, end_time). Read the description on the issues for an idea on how to build things.